Another Azure Platform upgrade

Chris reported `develop` CI failed this morning due to scheduled Azure brownout.  Looks like we need update the ubuntu images.

```
ubuntu-18.04 Python37


##[error]This is a scheduled Ubuntu-18.04 brownout. The Ubuntu-18.04 environment is deprecated and will be removed on December 1st, 2022. For more details, see https://github.com/actions/runner-images/issues/6002
,##[error]The remote provider was unable to process the request.
Pool: [Azure Pipelines](https://dev.azure.com/ComputationalCryoEM/1455a1f0-08c2-49c8-b076-83cebead3498/_settings/agentqueues?poolId=&queueId=26)
Image: ubuntu-18.04
```

My only concern is their naming is goofy...  They list 22.04, but still have 20.04 as `-latest`.  Makes me wonder if we should just always use specific versions. Maybe docs old. TBD.

https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml

--- a/azure-pipelines.yml
+++ b/azure-pipelines.yml
@@ -9,9 +9,9 @@ schedules:
 
 jobs:
 - job:
-  displayName: ubuntu-latest
+  displayName: ubuntu-22.04
   pool:
-    vmImage: 'ubuntu-latest'
+    vmImage: 'ubuntu-22.04'
   strategy:
     matrix:
       Python37:
@@ -31,9 +31,9 @@ jobs:
     displayName: pytest
 
 - job:
-  displayName: ubuntu-18.04
+  displayName: ubuntu-20.04
   pool:
-    vmImage: 'ubuntu-18.04'
+    vmImage: 'ubuntu-20.04'
   strategy:
     matrix:
       Python37:

